althea_kernel_interface/src/babel.rs

```diff
@@ -0,0 +1,7 @@
+use crate::KernelInterface;
+
+impl dyn KernelInterface {
+    pub fn setup_babel(&self) {
```

Comment:
please call this restart babel, more obvious, since there's a lot of babel setup rita actually does later we don't want to cause confusion.

Reply:
fixed